raise Exception("Nós geramos uma exceção!")
